README.md

```diff
@@ -2,31 +2,122 @@
+any expression that can evaluate to either "true" or "false".
+
+In this case, either the `grade` is greater than or equal to 70, or it's not.
+If it's greater than or equal to 70, then the program will execute the
+content of the `if` block. If the grade is less than 70, then the program will
+not execute the content of the `if` block. Consider the following flowchart:
+
+![flowchart](https://curriculum-content.s3.amazonaws.com/java-mod-1/if-statement/if-conditional-flowchart.png)
+
+Let's take a look at this control flow by running this program in the
+debugger. We'll put a breakpoint at the `if` statement and enter in the grade
+`90`:
+
+![if-statement-breakpoint](https://curriculum-content.s3.amazonaws.com/java-mod-1/if-statement/intellij-debugger-if-statement-breakpoint.PNG)
+
+As we can see in the debug console, the user entered 90 in as the input. If we
+take a look at the conditional clause, `grade >= 70` is equivalent to
+`90 >= 70`, which evaluates to `true`. Let's step-over to the next statement and
+see what happens in the execution:
+
+![execute-if-block](https://curriculum-content.s3.amazonaws.com/java-mod-1/if-statement/intellij-debugger-if-block-execution.PNG)
+
+As we predicted, the execution falls into the `if` block! Let's run this again,
+but this time, we'll have the user enter `60` instead:
+
+![if-statement-breakpoint-again](https://curriculum-content.s3.amazonaws.com/java-mod-1/if-statement/intellij-debugger-if-statement-breakpoint-2.PNG)
+
+When the user enters in 60, and we look at the conditional clause,
+`grade >= 70`, we can see that this is now equivalent to `60 >= 70`, which is
+`false`. Let's see what happens this time when we step-over to the next
+statement:
+
+![ignore-if-block](https://curriculum-content.s3.amazonaws.com/java-mod-1/if-statement/intellij-debugger-end-program.PNG)
+
+Since the grade entered was less than 70, the program does not execute the
+content of the `if` block and continues on with the program. In this case, there
+is nothing after the `if` statement, so it will jump to the end of the program.
+If we look at the console, we'll notice nothing was printed after prompting the
+user for a grade:
+
+```text
+Enter a numeric grade from 0 - 100:
+60
+```
+
+## Syntax of an `if` Statement
+
+An `if` statement can contain multiple statements inside it. Therefore, an `if`
+statement can be followed by an open curly brace `{` if it is meant to indicate
+the conditional execution of more than one statement. As always, the open curly
+brace `{` must then be matched with a close curly brace`}`:
+
+```java
+if (grade >= 70) {
+    // as many statements here that I need for my program
+}
+```
+
+Note that if we only have one conditional statement, we can omit the matching
+curly braces:
+
+```java
+if (grade >= 70)
+    System.out.println("Congrats! You passed!");
+```
+
+This notation is not recommended, as it makes it very easy to accidentally
```

Comment:
Good recommendation! When I first learned Java, I thought the goal was to make my files have as few lines as possible rather than to make them readable and less error-prone.